docs/ops/pooling/MaxPool_1.md

```diff
@@ -6,7 +6,79 @@
 
 **Short description**: [Reference](http://caffe.berkeleyvision.org/tutorial/layers/pooling.html)
 
-**Detailed description**: [Reference](http://cs231n.github.io/convolutional-networks/#pool)
+**Detailed description**: [Reference](https://deeplizard.com/learn/video/ZjM_XQa5s6s). Input shape can be either 3D, 4D or 5D. Pooling operation is performed with the respect to input shape from the third dimension to the last dimension. If paddings are used then during the pooling calculation their value is 0.  
+        `H_out = (H + pads_begin[0] + pads_end[0] - kernel[0] / strides[0]) + 1`  
+        `W_out = (H + pads_begin[1] + pads_end[1] - kernel[1] / strides[1]) + 1`  
+        `D_out = (H + pads_begin[2] + pads_end[2] - kernel[2] / strides[2]) + 1`  
+
+Example 1 shows how *MaxPool* operates with `auto_pad = explicit`
+
+```
+input = [[1, 2, 3],
+         [4, 5, 6],
+         [7, 8, 9]]
```

Comment:
Please change some values on the boundary to negative value and it will be more representable case